### PR TITLE
[16.0][FIX] account_avatax_sale_oca: Exemption is editable for Accounting Manager group

### PR DESCRIPTION
**Pull request by dreispt**
- [FIX] account_avatax_oca: Exemption is editable for Accounting Manager group
- [FIX] account_avatax_sale_oca: Exemption is editable for Accounting Manager group

### CODE REVIEW

**Comment by dreispt** [comment]
@tirth-osi Please review.

File: account_avatax_oca/views/account_move_view.xml [hunk 1]
```diff
@@ -17,16 +17,30 @@
...
+                <!-- Non editable Exemptions -->
                 <field
                     name="exemption_code"
                     readonly="1"
                     attrs="{'invisible': [('hide_exemption','=',True)]}"
+                    groups="!account.group_account_manager"
                 />
                 <field
                     name="exemption_code_id"
-                    readonly="1"
```
Comment (tirth-osi) [inline account_avatax_oca/views/account_move_view.xml:27]: @dreispt This exemption_code_id is for non account managers. So it is supposed to be read only.

Comment (tirth-osi) [reply account_avatax_oca/views/account_move_view.xml:27]: @dreispt Except for the 1 minor thing, everything else looks good to me.

**Comment by dreispt** [comment]
@tirth-osi fixed!

**Review by tirth-osi — APPROVED** [review]

**Pull request merged**